trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/system/table/GravitinoSystemTableCatalog.java

```diff
@@ -50,28 +47,25 @@ public Page loadPageData() {
 
     BlockBuilder nameColumnBuilder = VARCHAR.createBlockBuilder(null, size);
     BlockBuilder providerColumnBuilder = VARCHAR.createBlockBuilder(null, size);
-    MapBlockBuilder propertyColumnBuilder = STRING_MAPTYPE.createBlockBuilder(null, size);
+    BlockBuilder propertyColumnBuilder = VARCHAR.createBlockBuilder(null, size);
 
     for (GravitinoCatalog catalog : catalogs) {
       Preconditions.checkNotNull(catalog, "catalog should not be null");
 
       VARCHAR.writeString(nameColumnBuilder, catalog.getFullName());
       VARCHAR.writeString(providerColumnBuilder, catalog.getProvider());
-      Block mapValue =
-          buildMapValue(
-              STRING_MAPTYPE,
-              catalog.getProperties().size(),
-              (keyBuilder, valueBuilder) ->
-                  catalog
-                      .getProperties()
-                      .forEach(
-                          (key, value) -> {
-                            VARCHAR.writeString(keyBuilder, key);
-                            VARCHAR.writeString(valueBuilder, value);
-                          }));
-      STRING_MAPTYPE.writeObject(propertyColumnBuilder, mapValue);
+      try {
+        VARCHAR.writeString(
```

Comment:
So we change the data type from `Map` to `String`?

Reply:
Yes, The `MapBlockBuilder` interface has changed.